app/concepts/movies/representers/list.rb

```diff
@@ -12,7 +12,7 @@ def basic
           {
             id: movie.id,
             title: movie.title,
-            genre: movie.genre,
+            genre: movie.movie_genre,
```

Comment:
I would like to leave it as is because this movie_genre is a simple object with only id and a name. Unless you think that I should definitely make a representer for it? 🤔

Reply:
No problem, keep it as it is :ok_hand: